tools/download.py

```diff
@@ -34,20 +34,18 @@
 task_start = int(time.time())
 
 try:
-    r = subprocess.check_output(['download_ega_file.py','-p',project_code,'-f', ega_file_id+".aes", '-o', file_name])
+    if project_code in ['LINC-JP', 'BTCA-JP']:
```

Comment:
Do we really want to hardcode the projects here?

Maybe not now, but it should be possible to make it a bit more intelligent, maybe try first without 'path', if exception occurs due to missing file, then try with 'path'.